Update Go requirements (#2600)
pingcap/docs: pd-control.md

## Compile from source code

> 1. [Go](https://golang.org/) Version 1.13 or later since we're using go modules

Reviewer: ```suggestion
1. [Go](https://golang.org/) Version 1.13 or later because the Go modules are used.
```
